### ISSUE TITLE
Error with removing export values

### ISSUE THREAD

**Issue by sirenkovladd** [putoutjs/minify#22]
- **Version** (`npm -v`): 10.5.2
- **Node Version** `node -v`: v20.13.1
- **OS** (`uname -a` on Linux): Darwin arm64

Error with removing export values

```js
import { minify } from "@putout/minify";

const source = `function test() {
  return [1, 2];
}
export const [one, two] = test();`;

const result = minify(source);
console.log('>', result, '<');
```

```sh
❯ node i.js
>  <
```

**Comment by coderaiser** [comment]
The funny thing, that this is the same :)

```js
export const [one, two] = test();`;
```

and:

```js
const [one, two] = test();
export {
    one,
    two,
```

Just fixed in [v3.19.6](https://github.com/putoutjs/minify/releases/tag/v3.19.6) 🎉

**Comment by sirenkovladd** [comment]
Thank you
